autosklearn/data/target_validator.py

```diff
@@ -170,10 +170,8 @@ def _fit(
                 f" Data involved = {shape}/{self.type_of_target}"
             )
 
-        # Creat the encoder
-        self.encoder = OrdinalEncoder(
-            handle_unknown="use_encoded_value", unknown_value=-1
-        )
+        # Create the encoder
+        self.encoder = OrdinalEncoder(handle_unknown="error")
```

Comment:
So unknown values are now an Error where as before they were `-1`, seems like this would cause an issue unless we are sure everything is encoded before this point?

Reply:
This should not be an issue. This should only be called once in the beginning on all training data prior to any CV splitting etc.